src/sql/inspect.py

```diff
@@ -173,7 +174,7 @@ class Columns(DatabaseInspection):
     """
 
     def __init__(self, name, schema, conn=None) -> None:
-        util.is_table_exists(name, schema)
+        util._check_table_exists(name, schema)
```

Comment:
so I see you added logic here that will let this continue if name is defined as a snippet. I think that makes sense so this feature works but I wonder if it'll have any side-effects since we're using this class in multiple places

@yafimvo, thoughts?

Reply:
Hi, the `_check_table_exists` function first checks if the table is a snippet name, and if it is not we run the `is_table_exists` inside the earlier function. So, if we pass a table name that is neither a snippet or a table, this will work similar to the check we had before.